index.d.ts

```diff
@@ -88,6 +92,6 @@ declare module 'react-native-document-picker' {
       options: DocumentPickerOptions<OS>
     ): Promise<DocumentPickerResponse[]>;
     static isCancel<IError extends { code?: string }>(err?: IError): boolean;
-    static releaseSecureAccess(uris: Array<string>): void;
+    static pickDirectory(): Promise<DirectoryPickerResponse>;
```

Comment:
on ios this returns `null`, that should be reflected

Reply:
done